<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tableViewTools</class>
 <widget class="QWidget" name="tableViewTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>382</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>120</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>120</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>tableViewTools</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="styleSheet">
      <string notr="true">QToolBox
{
background-color: transparent ;
border : none ;
}
</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabSpacing">
      <number>0</number>
     </property>
     <widget class="QWidget" name="needed_tools">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>120</width>
        <height>340</height>
       </rect>
      </property>
      <attribute name="label">
       <string>nécessaire outils</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="toolButton_refresh">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
background-color: transparent ;
border : none ;
}
QToolButton::hover
{
	background-color: rgb(192,192,192) ;
}</string>
         </property>
         <property name="text">
          <string>Actualisé</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/Assets/Images/icons/refresh.svg</normaloff>:/Assets/Images/icons/refresh.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_Filter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
background-color: transparent ;
border : none ;
}
QToolButton::hover
{
	background-color: rgb(192,192,192) ;
}</string>
         </property>
         <property name="text">
          <string>Filter</string>
         </property>
         <property name="icon">
          <iconset resource="Qt_Resource_File/tableViewToolsResource.qrc">
           <normaloff>:/icon/icon/Filter Filled-100.png</normaloff>:/icon/icon/Filter Filled-100.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_suppr">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
background-color: transparent ;
border : none ;
}
QToolButton::hover
{
	background-color: rgb(192,192,192) ;
}</string>
         </property>
         <property name="text">
          <string>Supprimer 
cette ligne</string>
         </property>
         <property name="icon">
          <iconset resource="Qt_Resource_File/tableViewToolsResource.qrc">
           <normaloff>:/icon/icon/Delete-96.png</normaloff>:/icon/icon/Delete-96.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_view">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
background-color: transparent ;
border : none ;
}
QToolButton::hover
{
	background-color: rgb(192,192,192) ;
}</string>
         </property>
         <property name="text">
          <string>View</string>
         </property>
         <property name="icon">
          <iconset resource="Qt_Resource_File/tableViewToolsResource.qrc">
           <normaloff>:/icon/icon/Visible-96.png</normaloff>:/icon/icon/Visible-96.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="more_tools">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>120</width>
        <height>340</height>
       </rect>
      </property>
      <attribute name="label">
       <string>plus d'outils</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QToolButton" name="toolButton_resizeColumnsToContents">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
background-color: transparent ;
border : none ;
}
QToolButton::hover
{
	background-color: rgb(192,192,192) ;
}</string>
         </property>
         <property name="text">
          <string>resize Columns 
To Contents</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_clean_table">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
background-color: transparent ;
border : none ;
}
QToolButton::hover
{
	background-color: rgb(192,192,192) ;
}</string>
         </property>
         <property name="text">
          <string>clean Table</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_Export">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
background-color: transparent ;
border : none ;
}
QToolButton::hover
{
	background-color: rgb(192,192,192) ;
}</string>
         </property>
         <property name="text">
          <string>Export Table</string>
         </property>
         <property name="icon">
          <iconset resource="Qt_Resource_File/tableViewToolsResource.qrc">
           <normaloff>:/icon/icon/1469804931_Database-Upload.png</normaloff>:/icon/icon/1469804931_Database-Upload.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_Import">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton
{
background-color: transparent ;
border : none ;
}
QToolButton::hover
{
	background-color: rgb(192,192,192) ;
}</string>
         </property>
         <property name="text">
          <string>import Table</string>
         </property>
         <property name="icon">
          <iconset resource="Qt_Resource_File/tableViewToolsResource.qrc">
           <normaloff>:/icon/icon/1469804944_download-database.png</normaloff>:/icon/icon/1469804944_download-database.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
  <include location="Qt_Resource_File/tableViewToolsResource.qrc"/>
 </resources>
 <connections/>
</ui>
